asv_bench/benchmarks/utils/common.py

```diff
@@ -455,23 +455,30 @@ def trigger_import(*dfs):
     ----------
     *dfs : iterable
         DataFrames to trigger import.
+
+    Returns
+    -------
+    DataFrame or tuple of DataFrames if more than one object was passed
+        Returns exactly the same objects that were passed into the function.
```

Comment:
adding return here to ease variables initialization by already imported frames (see [here](https://github.com/dchigarev/modin/blob/7172e1fe9bc02649b008522b12d1a908c4b008d3/asv_bench/benchmarks/benchmarks.py#L519))